server/chat-plugins/trivia.ts

```diff
@@ -1659,14 +1603,14 @@ const commands = {
 			userid = toID(name);
 		}
 
-		const allTimeScore = triviaData.leaderboard[userid];
+		const allTimeScore = triviaData.leaderboard![userid];
 		if (!allTimeScore) return this.sendReplyBox(`User '${name}' has not played any trivia games yet.`);
-		const score = triviaData.altLeaderboard[userid] || [0, 0, 0];
+		const score = triviaData.altLeaderboard![userid] || [0, 0, 0];
 
 		const ranks = cachedAltLadder.get().ranks[userid];
-		const allTimeRanks = cachedLadder.get().ranks[userid];
-		const row = i => `<strong>${score[i]}</strong>${ranks ? ` (#${ranks[i]})` : ""}, ` +
-			`all time: <strong>${allTimeScore[i]}</strong> (#${allTimeRanks ? allTimeRanks[i] : allTimeRanks.length})<br />`;
+		const allTimeRanks: TriviaRank = cachedLadder.get().ranks[userid];
```

Comment:
I can't figure out a reason this can't be inferred.

```suggestion
		const allTimeRanks = cachedLadder.get().ranks[userid];
```